src/enableDisableRules.ts

```diff
@@ -81,7 +81,7 @@ export class EnableDisableRulesWalker extends SkippableTokenAwareRuleWalker {
                         // end at the beginning of the next line
                         this.enableDisableRuleMap[ruleToAdd].push({
                             isEnabled: !isEnabled,
-                            position: scanner.getTextPos() + 1,
+                            position: scanner.getTextPos(),
```

Comment:
Just thinking, maybe this would be a better way to get the beginning of the next line?
`position: this.getStartOfLinePosition(node, startingPosition, 1) - 1,`